draft-ietf-httpbis-incremental.md

```diff
@@ -124,6 +124,12 @@ intermediaries to understand and respect a request to deliver messages
 incrementally. Clients can rely on prior knowledge or probe for support on
 individual resources.
 
+The Incremental header field enables the establishment of a bidirectional byte
+channel over HTTP, as its presence in both requests and responses instructs
+intermediaries to forward early responses ({{Section 7.5 of HTTP}}) and to
+transmit message bodies incrementally in both directions.
```

Comment:
```suggestion
transmit message contents incrementally in both directions.
```
As pointed out by @Acconut in https://github.com/httpwg/http-extensions/pull/3066#discussion_r2053562127, we are operating on top of Semantics and should better be using "content."